examples/dynamodb-with-spark.ipynb

```diff
@@ -0,0 +1,158 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "59896e24",
+   "metadata": {},
+   "source": [
+    "## Connect DynamoDB using spark connector from EMR Studio Notebook using Spark Scala\n",
+    "\n",
+    "#### Topics covered in this example\n",
+    "\n",
+    "* Configuring the EMR-DynamoDB Connector\n",
+    "* Connecting to AWS DynamoDB using EMR-DynamoDB Connector to read data into Spark DF and get the count"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4e7434c0",
+   "metadata": {},
+   "source": [
+    "## Table of Contents:\n",
+    "\n",
+    "1. [Prerequisites](#Prerequisites)\n",
+    "2. [Introduction](#Introduction)\n",
+    "3. [Load the configuration in memory](#Load-the-configuration-in-memory)\n",
+    "4. [Read data using Scala](#Read-data-using-Scala)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "39997538",
+   "metadata": {},
+   "source": [
+    "## Prerequisites"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7329c0d3",
+   "metadata": {},
+   "source": [
+    " 1. EMR Version - emr-6.4.0\n",
+    " 2. The notebook is using Amazon EMR-DynamoDB Connector(emr-ddb-hadoop.jar) which is used to access data in Amazon DynamoDB using Apache Hadoop, Apache Hive, and Apache Spark in Amazon EMR. (https://docs.aws.amazon.com/emr/latest/ReleaseGuide/EMRforDynamoDB.html)\n",
+    " 3. The Amazon EMR-DynamoDB Connector is locally available within EMR at location below: \n",
+    " \n",
+    " `/usr/share/aws/emr/ddb/lib/emr-ddb-hadoop.jar`\n",
+    " 4. To locally use the spark.jars in the configuration using the `file://` protocol, Livy does not allow local files to be uploaded to the user session by default. So we need to update the livy configuration to allow the local directory and restart livy server to have these take effects.\n",
```

Comment:
Please explicitly mention that below commands should be run on the EMR cluster.

Reply:
Added a statement:
" ##### The following commands should be run from the EMR cluster attached to the notebook\n",